### PR TITLE
Fix website links in the Readme

### PR DESCRIPTION
**Pull request by breno12321**
## What has Changed?

Fix website links in the readme topics

image

## My Calva PR Checklist

I have:

- [X] Read [How to Contribute](https://github.com/BetterThanTomorrow/calva/wiki/How-to-Contribute#before-sending-pull-requests).
- [X] Directed this pull request at the `dev` branch. (Or have specific reasons to target some other branch.)
- [X] Made sure I have changed the PR base branch, so that it is not `published`. (Sorry for the nagging.)
- [ ] Updated the `[Unreleased]` entry in `CHANGELOG.md`, linking the issue(s) that the PR is addressing.
- [ ] Figured if **anything** about the fix warrants tests on Mac/Linux/Windows/Remote/Whatever, and either tested it there if so, or mentioned it in the PR.
- [ ] Added to or updated docs in this branch, if appropriate
- [ ] Tests
     - [ ] Tested the particular change
     - [ ] Figured if the change might have some side effects and tested those as well.
     - [ ] Smoke tested the extension as such.
     - [ ] Tested the VSIX built from the PR (so, after you've submitted the PR). You'll find the artifacts by clicking _Show all checks_ in the CI section of the PR page, and then _Details_ on the `ci/circleci: build` test.
- [ ] Referenced the issue I am fixing/addressing _in a commit message for the pull request_.
     - [ ] If I am fixing the issue, I have used [GitHub's fixes/closes syntax](https://help.github.com/en/articles/closing-issues-using-keywords)
     - [ ] If I am fixing just part of the issue, I have just referenced it w/o any of the "fixes” keywords.
- [ ] Created the issue I am fixing/addressing, if it was not present.
- [ ] Formatted all JavaScript and TypeScript code that was changed. (use the [prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or run `npm run prettier-format`)
- [ ] Confirmed that there are no linter warnings or errors (use the [eslint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), run `npm run eslint` before creating your PR, or run `npm run eslint-watch` to eslint as you go).

Ping @pez, @bpringe, @corasaurus-hex, @Cyrik

### CODE REVIEW

**Comment by breno12321** [comment]
Since its just a link fix I don't know if I have to do all the update change log tasks and so on 👀

File: README.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ _Distilled from [CIDER](https://cider.mx/)*._
...
-* inline [code evaluation](https://calva.io/evaluation.md)
-* [structural editing](https://calva.io/paredit.md)
-* code [formatting](https://calva.io/formatting.md)
+* inline [code evaluation](https://calva.io/evaluation/)
+* [structural editing](https://calva.io/paredit/)
+* code [formatting](https://calva.io/formatting/)
 * code navigation
-* a [debugger](ttps://calva.io/debugger.md)
-* [linting](https://calva.io/linting.md)
+* a [debugger](ttps://calva.io/debugger/)
```
Comment (bpringe) [inline README.md]:
```suggestion
* a [debugger](https://calva.io/debugger/)
```
Missed the `h` here. 😄

**Comment by bpringe** [comment]
Thanks for this! I added [an issue](https://github.com/BetterThanTomorrow/calva/issues/1701) for improving the PR template to explain what to do for docs-only changes.

File: README.md (hunk 1 above)
Comment (breno12321) [reply README.md]: Noicee

**Pull request merged**